CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ Main (unreleased)
 
  - It is now possible to compile Grafana Agent using Go 1.19. (@rfratto)
 
+ - Fix a bug in the Agent Operator CR quickstart guide (@hjet)
```

Comment:
IMO the changelog should only list user-facing code changes and not anything related to docs

Reply:
ack, will remove